src/Api/Middlewares/ExceptionMiddleware.cs

```diff
@@ -79,7 +81,12 @@ private async void HandleRequestValidationException(HttpContext context, Excepti
         };
         await context.Response.Body.WriteAsync(SerializeToUtf8BytesWeb(result));
     }
-    
+
+    private async void HandleLimitExceededException(HttpContext context, Exception ex)
```

Comment:
this could be mate static

Reply:
what do you mean?

Reply:
this method does not touch any class members and could be made static